lib/jekyll-import/importers/joomla3.rb

```diff
@@ -34,18 +35,24 @@ def self.process(options)
         host    = options.fetch('host', "localhost")
         cid	    = options.fetch('category', 0)
         table_prefix = options.fetch('prefix', "jos_")
+        state   = options.fetch('state', 1)
```

Comment:
Here, you'd receive `state = "unpublished"`, which you could then map into the integer value using a hash:

```ruby
STATE_NUMS = {
  "unpublished" => 0, 
  # etc
}
```